### DISCUSSION TITLE
Available a complete tutorial in Italian (Disponibile tutorial completo in Italiano)

### DISCUSSION THREAD

**Discussion by SurvivalHacking** [probonopd/MiniDexed#306] [General]
If you want to see a tutorial in Italian, search on youtube Survival Hacking you will find a video entitled 8 DX 7 in tasca.
I don't post link because I don't know if it is possible.

Se volete vedere un tutorial in Italiano, cercate su youtube   Survival Hacking  troverete un video dal titolo 8 DX 7 in tasca.
Non posto link perché non so se è possibile.

Bye

**Comment by probonopd** [comment]
Welcome @SurvivalHacking.
You made an impressive case.
Are you into coding? Maybe one could connect those LEDs to a i2c controller like the PCF8575 IO Expander Module and e.g., light up one LED for each Tone Generator if it is acivated, and make another blink if there is MIDI traffic affecting this Tone generator, or whatever the TX816 did.

Benvenuto @SurvivalHacking.
Il tuo caso è impressionante.
Sei appassionato di codifica? Forse si potrebbero collegare quei LED a un controller i2c come il PCF8575 IO Expander Module e, ad esempio, accendere un LED per ogni generatore di toni se è attivo, e farne lampeggiare un altro se c'è traffico MIDI che interessa questo generatore di toni, o qualsiasi altra cosa facesse il TX816.

![image](https://user-images.githubusercontent.com/2480569/178474456-b74f5d29-b2ad-4280-ae4e-358446f4831f.png)
